src/data_info/services/metadata.clj

```diff
@@ -1,11 +1,13 @@
 (ns data-info.services.metadata
   (:use [clojure-commons.error-codes]
         [clojure-commons.validators]
-        [clj-jargon.item-ops :only [copy-stream input-stream]]
+        [clj-jargon.item-ops :only [copy-stream input-stream output-stream]]
```

Comment:
I had to modify `output-stream` to be public. I didn't see any problem with doing that, but someone with more experience with `clj-jargon` should probably verify that that's the case.